src/platforms/php/guides/symfony/profiling/index.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Profiling
+description: 'Learn more about how to configure our Profiling integration and start profiling your code.'
+---
+
+<Note>
+
+Profiling for PHP is supported from Sentry Symfony SDK version `4.7.0` and above.
+
+</Note>
+
+With [profiling](/product/profiling/), Sentry allows you to collect and analyze performance profiles from real users in production to give you a complete picture of how your application performs in a variety of environments.
+
+Under the hood, we're using Wikipedia's sampling profiler [Excimer](https://github.com/wikimedia/mediawiki-php-excimer).
+The Excimer PHP extension does support PHP 7.2 and up, for PHP 8.2 support, make sure to use Excimer version 1.1.0.
+
+**Excimer requires Linux and does not support Windows or macOS.**
+
+## Installation
+
+### Linux package manager
+
+```bash
+apt-get install php-excimer
+```
+
+See https://www.mediawiki.org/wiki/Excimer for additional distributions.
+
+### PECL
+
+```bash
+pecl install excimer
+```
+
+See https://pecl.php.net/package/excimer for more information.
+
+### Build from source
+
+```bash
+git clone https://github.com/wikimedia/mediawiki-php-excimer.git
+
+cd excimer/
+phpize && ./configure && make && sudo make install
+```
+
+Depending on your environment, you may need to enable the Excimer extension afterwards.
+
+```
+phpenmod -s fpm excimer
+# or
+phpenmod -s apache2 excimer
+```
+
+## Enable Performance Monitoring
+
+Profiling depends on Sentry's performance monitoring product being enabled beforehand.
+Check out the <PlatformLink to="/performance/">performance setup</PlatformLink> and <PlatformLink to="/performance/instrumentation/custom-instrumentation/">custom instrumentation</PlatformLink> documentation for more detailed information.
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+    options:
+        traces_sample_rate: 1.0
+```
+
+## Enabling Profiling
+
+```yaml {filename:config/packages/sentry.yaml}
+sentry:
+    options:
+        traces_sample_rate: 1.0
+        # Set a sampling rate for profiling - this is relative to traces_sample_rate
+        profiles_sample_rate: 1.0
+```
+
+## Improve Response Time
+
+Using the profiling capabilities in your Symfony application may has some impact on your overall response time (depending on the `traces_sample_rate` you configured).
+If your web server is using FastCGI, our SDK utilizes the [`kernel.terminate` event](https://symfony.com/doc/current/components/http_kernel.html#8-the-kernel-terminate-event), which means that the response is sent to the user before the profiling data is sent to Sentry.
+
+If your web server is not using FastCGI, you can use [Relay](/product/relay/getting-started/) running locally on the same machine or local network that acts as a proxy/agent. As a result, instead of your PHP process sending data to Sentry, the PHP process sends it to your local Relay, which then forwards it to Sentry.
```

Comment:
```suggestion
If your web server isn't using FastCGI, you can run [Relay](/product/relay/getting-started/) locally on the same machine or a local network that can act as a proxy/agent. Doing this will make the PHP process send requests to your local Relay, which will then forward them to Sentry, instead of sending requests to Sentry directly.
```